### DISCUSSION TITLE
Window opening and closing

### DISCUSSION THREAD

**Discussion by ExtF8** [Schneegans/Burn-My-Windows#436] [Ideas]
Hey,

Thank you for the amazing extension.

Maybe I am mistaken, but in previous versions, I could choose a different animation for opening and closing the window. Now, I see that it is random. I would love to have the choice to choose. I am willing to try and re-implement this functionality, if you could point me in the right direction, as I am a newbie in software development.

**Comment by Schneegans** [comment]
This is still possible. Just checkout the effect profiles at the top of the preferences window! There you can create different effect configurations and choose in detail when to apply which.

**Reply by ExtF8** [reply]
Oh, okay. I see now. It is a bit confusing, but thanks, I get it now.

**Comment by augmentedagent** [comment]
Is the preferences configuration available in the Kwin version? I only see each effect in the "desktop effects" of system settings, where you configure each effect individually with no ability to set it for "use upon closing windows only".

**Reply by Schneegans** [reply]
For KWin, this is currently not possible. See this: https://github.com/Schneegans/Burn-My-Windows/issues/173
